File: com/pdfjet/InitialState.java
package com.pdfjet;

public class InitialState {
    public boolean visible = false;
    public boolean printable = false;
    public boolean exportable = false;

    public InitialState setVisible(boolean visible) {
        this.visible = visible;
        return this;
    }

    public InitialState setPrintable(boolean printable) {
        this.printable = printable;
        return this;
    }

    public InitialState setExportable(boolean exportable) {
        this.exportable = exportable;
        return this;
    }
}

File: net/pdfjet/InitialState.cs
using System;
using System.Collections.Generic;

namespace PDFjet.NET {
public class InitialState {
    public bool visible = true;
    public bool printable = false;
    public bool exportable = false;

    public InitialState SetVisible(bool visible) {
        this.visible = visible;
        return this;
    }

    public InitialState SetPrintable(bool printable) {
        this.printable = printable;
        return this;
    }

    public InitialState SetExportable(bool exportable) {
        this.exportable = exportable;
        return this;
    }
}
}